src/Parser/Priest/Shadow/Modules/Items/Tier21_4set.js

```diff
@@ -28,17 +29,23 @@ class Tier21_4set extends Analyzer {
     const spellId = event.ability.guid;
     const combatant = this.combatants.selected;
 
-    if (spellId !== SPELLS.MIND_BLAST.id && spellId !== SPELLS.MIND_FLAY.id) return;
+    if (spellId !== SPELLS.MIND_BLAST.id && spellId !== SPELLS.MIND_FLAY.id && spellID !== SPELLS.VOID_BOLT.id ) return;
     if (!combatant.hasBuff(SPELLS.VOIDFORM_BUFF.id)) return;
 
     const { stacks: currentVoidformStacks } = combatant.getBuff(SPELLS.VOIDFORM_BUFF.id);
     const increasedCritChance = currentVoidformStacks * SET_INCREASE_CRIT_CHANCE_PER_VOIDFORM_STACK / 100;
 
     const critChanceWithoutSet = 1 + combatant.critPercentage;
     const critChanceWithSet = increasedCritChance + critChanceWithoutSet;
-
+    
+    if( spellID !== SPELLS.VOID_BOLT.id )
+    {
     const hitDamage = event.hitType === HIT_TYPES.CRIT ? (calculateEffectiveDamage(event, 1) / CRIT_TO_HIT_MODIFIER) : (calculateEffectiveDamage(event, 1));
-
+    }
+    else
+    {
+    const hitDamage = event.hitType === HIT_TYPES.CRIT ? (calculateEffectiveDamage(event, 1) / CRIT_TO_HIT_MODIFIER_VOIDBOLT) : (calculateEffectiveDamage(event, 1));
+    }
```

Comment:
you have two const called hitDamage, probably just declare them higher up as variables in the class and then alter them here.